### PR TITLE
1.10 clang 13 format #1933

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

File: .github/workflows/clang-format-fix.yml [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-      uses: DoozyX/clang-format-lint-action@v0.11
+    - name: Run clang-format style check for C and Java programs.
+      uses: DoozyX/clang-format-lint-action@v0.13
       with:
         source: '.'
-        extensions: 'c,h,cpp,hpp'
-        clangFormatVersion: 10
-        inplace: True
+        extensions: 'c,h,cpp,hpp,java'
+        clangFormatVersion: 13
```
Comment (lrknox) [inline .github/workflows/clang-format-fix.yml:17]: develop, 1.12 and 1.8 PRs have a line "inplace:  True".  Should that be here for 1.10 as well?

Comment (byrnHDF) [reply .github/workflows/clang-format-fix.yml:17]: probably

Comment (byrnHDF) [reply .github/workflows/clang-format-fix.yml:17]: yes that file got the wrong changes

Comment (derobins) [reply .github/workflows/clang-format-fix.yml:17]: When this is fixed we can merge this

**Review by derobins — APPROVED** [review]

**Review by lrknox — APPROVED** [review]

**Pull request merged**